Resources/Prototypes/_Goobstation/Entities/Clothing/OuterClothing/specific.yml

```
@@ -37,8 +39,13 @@
     clothingPrototypes: 
       head: ClothingHeadHelmetHEV
   - type: ClothingGrantComponent
-    component:
+    component: # same as CE suit
       - type: SupermatterImmune
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)